functions_battery.sh

```diff
@@ -43,5 +43,9 @@ function __batt_pct() {
 # now, as a function, we can easily handle a conditional
 function __batt_time() {
   btime=$(pmset -g batt | grep -Eo '([0-9][0-9]|[0-9]):[0-5][0-9]')
-  if [[ "$btime" == "0:00" ]]; then echo ''; else echo " [$btime]"; fi
+  etime=$(pmset -g batt | grep -Eo 'estimate')
+  if [[ "$etime" == "estimate" ]]; then echo ", estimating"; else
+  if [[ "$btime" == "0:00" ]]; then echo ''; else echo ", $btime left"; fi; fi
 }
+
+export PS1="[\$(__batt_pct), \$(__batt_state)\$(__batt_time)] [\d \t] $ "
```

Comment:
I'd prefer if you made this a commented-out example line.  Putting it in here is highly opinionates this library, as it forces everyone to have a prompt line that's formatted this way, instead of letting people use the functions in whatever formatting they want.  That said, i like the addition of an example in the code, in case they don't read the same example in the `README.md` file.